table/tables/cache.go

```diff
@@ -30,74 +35,158 @@ var _ table.CachedTable = &cachedTable{}
 
 type cachedTable struct {
 	TableCommon
-	kv.MemBuffer
-	mu sync.RWMutex
+	cacheData atomic.Value
+	mu        sync.RWMutex
```

Comment:
mutex will not needed. I think cacheData set and get operations must be atomic operations

Reply:
I think it's safer to use a mutex instead of atomic. It seems that the happens before behavior for atomic is not documented even though it works well in practice...  see: https://stackoverflow.com/questions/42231153/do-atomic-operations-establish-a-happen-before-relation

Reply:
ok, I found golang's document also use `atomic.Value` in this way ... https://pkg.go.dev/sync/atomic#example-Value-Config